.github/workflows/docker-image.yml

```diff
@@ -0,0 +1,42 @@
+name: Build Docker Images & Push to Registry
+
```

Comment:
I assume this is a cloudbuild yaml? I would leave cloudbuild in the name somewhere if so